db/seeds/questions.yml

```diff
@@ -174,6 +240,12 @@ why_no_retreat:
   question_type: long_text
   required: false
 
+fa20_heard_from:
+  title: How did you first hear about Blueprint?
+  hint: Please let us know how you first heard about Blueprint, whether it’s through social media, through a friend, through classes, through a certain Sproul-alternative, etc.
```

Comment:
I'd cut the first 'Please let us know how you first heard about Blueprint' bc it's rather redundant.